src/components/Navbar/index.tsx

```diff
@@ -25,28 +38,32 @@ export const Navbar: React.FC = () => {
             <StyledSubtitle>Evergreen</StyledSubtitle>
           </Logo>
         </Link>
+        {legacyURL && (
+          <StyledLink href={legacyURL}>Switch to legacy UI</StyledLink>
```

Comment:
The contrast ratio on this is too low. Can you change the foreground color to something in the light spectrum? We can probably use the same white as the Evergreen title
<img width="338" alt="Screen Shot 2020-06-05 at 9 43 32 AM" src="https://user-images.githubusercontent.com/29384396/83883281-4ad55900-a711-11ea-8eac-50f17aec2ab8.png">